sidecar/sokuji_sidecar/accel.py

```
@@ -129,14 +183,18 @@ def probe(force: bool = False) -> Machine:
     apple = _safe(_apple_silicon, False)
     dml = _safe(_dml_adapters, ())
     installed = _safe(_installed, frozenset())
+    tc_kinds = _safe(_tc_kinds, ())
+    tc_gpus = _safe(_tc_gpus, ())
     fp_src = (f"{platform.system()}|{platform.machine()}|{int(apple)}|"
               f"{','.join(sorted(dml))}|{','.join(sorted(installed))}|"
+              f"{','.join(tc_kinds)}|"
+              f"{','.join(f'{k}:{n}:{t}' for k, n, t in tc_gpus)}|"
               f"{len(nvidia)}:{','.join(g.name for g in nvidia)}")
     fp = hashlib.sha1(fp_src.encode()).hexdigest()[:12]
```

Ruff (0.15.20)

[error] Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)